omnigibson/utils/motion_planning_utils.py

```diff
@@ -510,6 +519,13 @@ def overlap_callback(hit):
 
         valid_hit = hit.rigid_body not in context.disabled_collision_pairs_dict[mesh_path]
 
+        # if an non-sampling pose collides with any object, raise an action primitive error with the colliding object and robot mesh_path
```

Comment:
Don't mention the sampling thing. Just say output a warning if verbose mode is on and ... yada yada